### PR TITLE
GameDB: Add missing PAL demos

### PR DESCRIPTION
**Pull request by JordanTheToaster**
### Description of Changes
Adds some missing PAL demos to the DB.

### Rationale behind Changes
Missing entry's are bad.

### Suggested Testing Steps
Make sure CI is happy.

### CODE REVIEW

**Review by Mrlinkwii — APPROVED** [review]

**Review by RedDevilus — APPROVED** [review]
PCSX2Verse grows with these extra serials.

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -1765,6 +1765,9 @@ SCED-50381:
 SCED-50404:
   name: "Official PlayStation 2 Magazine Demo 10" # German
   region: "PAL-E-G"
+SCED-50450:
+  name: "Le Monde des Bleus 2002"
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:1769]: shouldn't this have [Demo] on the end?

Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:1769]: image

Seems like it.

Comment (JordanTheToaster) [reply bin/resources/GameIndex.yaml:1769]: _You see noooothing_

**Pull request merged**